pkgs/tools/misc/woeusb/default.nix

```diff
@@ -36,7 +36,7 @@ stdenv.mkDerivation rec {
     # should be patched with a less useless default PATH, but for now
     # we add everything we need manually.
     wrapProgram "$out/bin/woeusb" \
-      --set PATH '${stdenv.lib.makeBinPath [ coreutils dosfstools findutils gawk gnugrep grub2_light ncurses ntfs3g parted utillinux wget ]}'
+      --set PATH '${stdenv.lib.makeBinPath [ coreutils dosfstools findutils gawk gnugrep grub2_light ncurses ntfs3g parted utillinux wget p7zip ]}'
```

Comment:
I would prefer it if you make it clear in the expression that `p7zip` is an optional dependency. I don't mind it being enabled by default.

That would mean an `withWin7EfiSupport ? true` or something like that argument and using `++ lib.optionals withWin7EfiSupport [ p7zip ]` together with a comment explaining what you did in the PR message (what 7z is needed for) here.